src/migrations/meta/_journal.json

```diff
@@ -64,6 +64,20 @@
       "when": 1756985479294,
       "tag": "0008_organic_agent_zero",
       "breakpoints": true
+    },
```

Comment:
Why is there 2 snapshots here. Pretty sure it's not right.

You want to go back to main and restart from scratch then replay the generate from main state.

Here there are 2 migrations in rows related to the way you did the work